executor/src/main/java/com/redhat/service/bridge/executor/logging/ExecutorLoggingJsonProvider.java

```diff
@@ -0,0 +1,26 @@
+package com.redhat.service.bridge.executor.logging;
+
+import java.io.IOException;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import org.jboss.logmanager.ExtLogRecord;
+
+import com.redhat.service.bridge.infra.logging.CustomerIdProvider;
+
+import io.quarkiverse.loggingjson.JsonGenerator;
+import io.quarkiverse.loggingjson.JsonProvider;
+
+@ApplicationScoped
+public class ExecutorLoggingJsonProvider implements JsonProvider {
```

Comment:
Done. In order to support this for the `Executor` I had to pass the `customerId` as an environment variable (otherwise I needed to extract it from the `Processor` definition which would mean I had to keep the code based approach).